code_of_conduct.md

```diff
@@ -0,0 +1,128 @@
+# Hypertrace Code of Conduct
```

Comment:
@dnielsen is this the content you wanted here? Just to verify before it is copied around

Reply:
@dnielsen , can you please check COC at https://github.com/hypertrace/hypertrace/blob/adding-contributing/.github/code_of_conduct.md and confirm this is what we want?